### PR TITLE
robot-name: remove dead link

### PR DESCRIPTION
**Pull request by cmcaine**
gSchool.it is now a domain parking spot

### CODE REVIEW

**Comment by rpottsoh** [comment]
@iHiD, it is strange that GitHub is still expecting for the check to be completed but when I check Jenkins CI I see that the build was green.  In a situation like this usually something went wrong with the build and I can just restart it and everything works itself out.  This time I am not seeing a way to restart the build (assuming doing so will "fix" this).

File: exercises/robot-name/metadata.yml [hunk 1]
```diff
@@ -1,4 +1,3 @@
 ---
 blurb: "Manage robot factory settings."
 source: "A debugging session with Paul Blackwell at gSchool."
-source_url: "http://gschool.it"
```
Comment (rpottsoh) [inline exercises/robot-name/metadata.yml:4]: This line maybe should be
```
source_url: ""
```
But I am not positive.

**Comment by iHiD** [comment]
Closed and reopened the PR to restart the build.

**Comment by iHiD** [comment]
@rpottsoh Looks like this has worked :)

**Comment by rpottsoh** [comment]
@ihid, indeed.  I was looking for some documentation on the metadata.yml file and I am not finding anything.  @cmcaine's change may be fine as is, I just am not sure....

In any event I don't think this PR should be blocked.

**Comment by iHiD** [comment]
I'm unsure.

**Comment by SaschaMann** [comment]
There are various exercises without it (GH search only finds 98 files with `source_url` in this repo), so it should be fine, I think. E.g. [all-your-base](https://github.com/exercism/problem-specifications/blob/master/exercises/all-your-base/metadata.yml).

**Comment by ErikSchierboom** [comment]
I'm almost certain the `source_url` can be left out.

**Pull request merged**